src/main/java/drukier/maze/MazeDraw.java

```diff
@@ -0,0 +1,62 @@
+package drukier.maze;
+
+import javax.swing.*;
+import java.awt.*;
+
+public class MazeDraw extends JComponent {
+
+    Maze myMaze;
+    int width;
+    int height;
+
+    int lineWidth;
```

Comment:
LineWidth and lineHeight should be double, not interested, because you multiply it later.